fuel-vm/src/interpreter/memory.rs

```diff
@@ -294,15 +319,9 @@ pub(crate) fn load_byte(
     b: Word,
     c: Word,
 ) -> Result<(), RuntimeError> {
-    let bc = b.saturating_add(c) as usize;
-
-    if bc >= VM_MAX_RAM as RegisterId {
-        Err(PanicReason::MemoryOverflow.into())
-    } else {
-        *result = memory[bc] as Word;
-
-        inc_pc(pc)
-    }
+    let range = MemoryRange::new_overflowing_op(Word::overflowing_add, b, c, 1u64)?;
```

Comment:
```suggestion
    let range = MemoryRange::new_overflowing_op(Word::overflowing_add, b, c, 0u64)?;
```

Reply:
We are writing a single byte, why should this be `0`? That would mean an empty range, which then would not include the byte we're writing to.